common/include/pcl/conversions.h

```diff
@@ -254,7 +254,10 @@ namespace pcl
     // Fill point cloud binary data (padding and all)
     size_t data_size = sizeof (PointT) * cloud.points.size ();
     msg.data.resize (data_size);
-    memcpy (&msg.data[0], &cloud.points[0], data_size);
+    if (data_size)
```

Comment:
I'd write data_size > 0 to be clear what's happening although it's equivalent since it's of type size_t.